raiden/tests/scenarios/README.md

```diff
@@ -27,6 +27,13 @@ It is also tested that a node can be stopped and started again and that it still
 Several nodes perform up to 100 payments.
 In the end it is tested that channels can be closed and that the monitoring service correctly kicks in if a node is offline during closing.
 
+#### [bf2_long_running](./bf2_long_running.yaml)
+
+This is the bf2 long running scenario. This scenario mimics user behaviour for 
```

Comment:
Please, delete the first sentence here in order to be consistent with the other descriptions.